Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/controllers/HealthController.java

```diff
@@ -5,7 +5,7 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/_health")
+@RequestMapping("/health")
```

Comment:
Do we have an project-wide convention to put this at `/health` rather than `/_health`?  `SecurityHelper` seems to assume it's at `/_health`.  Regardless, should probably put the correct value in a constant and reference that where needed.